src/screens/browser/components/ModalAccount/index.tsx

```diff
@@ -0,0 +1,206 @@
+// Create sample react native template component
+import React, { useContext } from 'react';
+import { scale } from 'device';
+import Modal from 'react-native-modal';
+import { useSelector } from 'react-redux';
+import { View, StyleSheet, Text, Platform, ActivityIndicator, ScrollView } from 'react-native';
+
+import { IconCircleClose, colors, textStyles } from 'assets';
+import { getwebUrl } from 'utils/selectors';
+import { getBase64IdentIcon } from 'utils/helpers/identicon';
+import { CButton } from 'components';
+import { AccountInfo, useMyAccounts } from 'utils/hooks/useAccountInfo';
+import AccountItem from './AccountItem';
+import CTextButton from 'components/CTextButton';
+import { useConnectedSite } from 'screens/browser/hooks/useConnectedSite';
+import { useUpdateAccountConnectedSite } from 'screens/browser/hooks/useUpdateAccountConnectedSite';
+import { useDisconnectWithAccount } from 'screens/browser/hooks/useDisconnectWithAccount';
+import { useDisconnectAllAccounts } from 'screens/browser/hooks/useDisconnectAllAccounts';
+import { useConnectWithAccount } from 'screens/browser/hooks/useConnectWithAccount';
+import { getUrlWithProtocol } from 'screens/browser/utils/url';
+import BrowserContext from 'screens/browser/context';
+
+type Props = {
+  isVisible: boolean;
+  onClose?: () => void;
+};
+
+const ModalAccount = ({ isVisible, onClose }: Props) => {
+  const webUrl = useSelector(getwebUrl);
+  const browserRef = useContext(BrowserContext);
+  const { data: accounts, isLoading } = useMyAccounts({
+    enabled: isVisible,
+    onError: (err) => {
+      console.log('err: ', err);
```

Comment:
need to send error to sentry and set `useErrorBoundary=true` to fallback to error page